### PR TITLE
Add GitHub actions to dependabot

### PR DESCRIPTION
**Pull request by murny**
- Adds dependabot for github action packages
- Slight fixes for dependabot config and prettier reformat
- update github actions config

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
+    schedule:
+      interval: daily
+      time: "10:00"
+      timezone: America/Edmonton
+    open-pull-requests-limit: 10
+    target-branch: integration
+    labels:
+      - "dependencies"
+      - "ruby"
+  - package-ecosystem: github-actions
```
Comment (murny) [inline .github/dependabot.yml:40]: Most of this file is prettier changes.

This is pretty much the change here which adds github actions to dependabot upgrading.

I also explicitly added labels and timezone to the other packages (npm/rubygems)

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -21,21 +21,8 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
```
Comment (murny) [inline .github/workflows/push.yml:24]: This one line basically does what the line 25 to 38 diff was doing.

File: .github/dependabot.yml (hunk 1 above)
Comment (pgwillia) [reply .github/dependabot.yml:40]: Nice work!  Dependabot will update lines like https://github.com/ualbertalib/jupiter/blob/371be28a31f27847dc3424f21fbe9a06d28b032c/.github/workflows/push.yml#L13

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-actions-up-to-date-with-dependabot#enabling-dependabot-version-updates-for-actions

File: .github/workflows/push.yml (hunk 2 above)
Comment (pgwillia) [reply .github/workflows/push.yml:24]: Another nice catch! 

https://github.com/actions/cache/blob/main/examples.md#ruby---bundler

**Review by pgwillia — APPROVED** [review]

**Pull request merged**